### ISSUE TITLE
Fix accessibility issues with breadcrumbs component

### ISSUE THREAD

**Issue by certainlyakey** [e-gov/cvi#82]
### What happened?

The breadcrumbs component lacks accessibility (and usability partially is also hurt) - `href` attribute is missing (see eg. [here](https://www.tempertemper.net/blog/links-missing-href-attributes-and-over-engineered-code) why this is an issue). The links in the breadcrumbs must have URLs provided. 

As a part of this issue all necessary optimisations allowing for the best experience in a screen reader should be done (based on testing with screen reader software). Eg. nav or aria-label could help with the correct comprehension of the component in assistive devices as a "proper" breadcrumbs widget. 

### Reproduction steps

1. Go to https://e-gov.github.io/cvi/?path=/story/angular-breadcrumbs--default
2. Check the generated markup
3. Expected: links have `href`s
4. Observed: `href`s are absent

### Package

Angular

### Version

1.8.0

### Reproduction URL

https://e-gov.github.io/cvi/?path=/story/angular-breadcrumbs--default

**Comment by certainlyakey** [comment]
@ekateriinal is working on it.
